README.rst

```diff
@@ -8,7 +8,24 @@ A Python client library for working with the DSpace API. This was written agains
 Usage
 -----
 
```

Comment:
I would also include information on how to install the library from git using pip/pipenv. The command for pipenv is `pipenv install "git+https://github.com/mitlibraries/dspace-python-client.git@<release-version-number>#egg=dspace"`.